packages/nimble-components/src/rich-text/viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -2,13 +2,43 @@ import { html } from '@microsoft/fast-element';
 import { RichTextViewer, richTextViewerTag } from '..';
 import { fixture, type Fixture } from '../../../utilities/tests/fixture';
 import { RichTextViewerPageObject } from '../testing/rich-text-viewer.pageobject';
+import { anchorTag } from '../../../anchor';
+import { waitForUpdatesAsync } from '../../../testing/async-helpers';
+import {
+    RichTextMentionUsers,
+    richTextMentionUsersTag
+} from '../../../rich-text-mention/users';
+import { richTextMentionUsersViewTag } from '../../../rich-text-mention/users/view';
+import { MappingUser, mappingUserTag } from '../../../mapping/user';
 
 async function setup(): Promise<Fixture<RichTextViewer>> {
     return fixture<RichTextViewer>(
         html`<nimble-rich-text-viewer></nimble-rich-text-viewer>`
     );
 }
 
+async function appendUserMentionConfiguration(
```

Comment:
I think we should have some test coverage for:
1. mention users are supplied via template rather than appended later
2. multiple mapping users

Reply:
Added the above mentioned tests in rich-text-viewer.spec.ts